bigtop.mk

```diff
@@ -293,10 +293,10 @@ IMPALA_NAME=impala
 IMPALA_RELNOTES_NAME=Impala
 IMPALA_PKG_NAME=impala
 IMPALA_BASE_VERSION=2.2.0
-IMPALA_PKG_VERSION=$(IMPALA_BASE_VERSION)+hdp2.3.0+rocana1.3.0
+IMPALA_PKG_VERSION=$(IMPALA_BASE_VERSION)+hdp2.3.0+rocana1.4.0
 IMPALA_RELEASE_VERSION=1
```

Comment:
Thanks @joey.
Some extra info came out: @splatch found the commit on the rc1 branch where we did use `0.rc1` in the version number (whew!). So this PR is good.